server/api/src/requirements.txt

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
This file shouldn't be committed if it's auto-generated and removed by `make clean`.

Reply:
Oh wait, deleting requirements during clean is probably done by mistake.

The idea behind attaching the `requirements.txt` into codebase is that person with only docker installed is able to quickly run the app. WDYT? 

Drawback of this approach lies in pipenv which does not play nicely with docker... so `requirements.txt` is redundant way to patch docker path... On the other hand pipenv simplifies creation of the `requirements` file and can be scripted with make ... without it one needs to remember to activate the venv first ... I don't know much about python so maybe I go in circles here.

Reply:
How about using a multi-stage Docker build and generating requirements in the builder?

Reply:
Do you mean builder image installs `pipenv` to generate `requirements.txt` and then uses `pip` to install deps ❓ 

Maybe I should revert to use just plain `pip` & `requirements.txt` to manage the dependencies, so no python is needed if you want to run the server in the container and we don't track the dependencies in multiple places 🤔

Reply:
:+1: for reverting to plain pin.

Reply:
Done in #3b86010